setup/README.adoc

```diff
@@ -60,18 +61,29 @@ Note that the template does not take any parameter, and all resources will be cr
 
 == Dev Sandbox Setup
 
-1. Clone this repository +
+. Clone this repository +
 `git clone git@github.com:codeready-toolchain/toolchain-e2e.git`
-2. Run the following to install the Dev Sandbox operators +
+. Run the following to install the Dev Sandbox operators +
 `make dev-deploy-e2e`
-3. Populate the cluster with users (by default the tool will provision 3000 users) and create resources based on the user-workloads.yaml file you created in the prereqs step above. +
+. Run `oc get toolchainstatus -n <quay-io-username>-host-operator` and ensure the overall status is `Ready`
++
+```
```

Comment:
Well... oc get toolchainstatus (without -o yaml) won't give you such details like the actual condition. It will show you the overall column STATUS where you can expect READY.

Reply:
Maybe worth adding a screenshot to avoid confusions?

Reply:
![Screenshot from 2021-03-24 15-11-45](https://user-images.githubusercontent.com/620087/112390497-a7f7c680-8cb3-11eb-9b7e-047d5cad5e0f.png)

Reply:
Oops, I copy-pasted from the chat haha. I missed that it didn't have the `-o yaml` but actually without it is even better. I updated the text to just show 
```
NAME               MURS   READY   LAST UPDATED
toolchain-status   0      True    2021-03-24T22:39:36Z
```

I guess that should be clear enough

Reply:
> I updated the text to just show

Have you? I don't see any changes.

Reply:
Ah, I created the PR using the github UI because it's convenient to modify the README and preview it but now that I have owner privileges it created the branch on origin. I didn't realize that. For the edits I pulled the PR locally and then pushed the changes to my fork. Let me delete the branch on origin and create a PR from my fork.

Reply:
Actually, I pushed the change to this PR since we have the comments already. I will just delete this branch from origin after merging the PR.